Source/ExecutionEngine/ExecutionEngine.cs

```diff
@@ -1331,7 +1331,7 @@ private static void UpdateStatistics(PipelineStatistics stats, VC.VCGen.Outcome
           continue;
         }
 
-        var errorInfo = error.CreateErrorInformation(outcome, Options.ForceBplErrors);
+        var errorInfo = error.CreateErrorInformation(vcOutcome, Options.ForceBplErrors);
```

Comment:
I'm reading all these vcOutcome, since you are doing refactoring, would it make sense to remove the abbreviation and name this "verificationConditionOutcome" or just "verificationOutcome" ?

Reply:
I'm not sure about that rename. I'll leave it for another time.